hydra_base/lib/units.py

```diff
@@ -239,6 +259,16 @@ def get_dimension_by_name(dimension_name,**kwargs):
         raise ResourceNotFoundError("Dimension %s not found"%(dimension_name))
 
 
+def get_default_dimension_id(**kwargs):
+    """
+        This method returns the id the default dimension name (ex "dimensionless")
+    """
+    default_dimension = db.DBSession.query(Dimension).filter(Dimension.is_default=='Y').first()
```

Comment:
Didn't we agree to remove the is_default column?

Reply:
just a refuse to delete. removed it